src/index.js

```diff
@@ -19,16 +19,51 @@ export default declare(({
 }) => {
   assertVersion(7);
 
-  const buildSvg = template(`
-  var SVG_NAME = function SVG_NAME(props) { return SVG_CODE; };
-`);
-
-  const buildSvgWithDefaults = template(`
-  var SVG_NAME = function SVG_NAME(props) { return SVG_CODE; };
-  SVG_NAME.defaultProps = SVG_DEFAULT_PROPS_CODE;
-`);
+  const buildSvg = ({
+    IS_EXPORT, EXPORT_FILENAME, SVG_NAME, SVG_CODE,
```

Comment:
If the entire signature can’t go on one line, each argument needs to go on it’s own line.
```suggestion
    IS_EXPORT,
    EXPORT_FILENAME,
    SVG_NAME,
    SVG_CODE,
```